iphone/Classes/ContactsModule.h

```diff
@@ -6,7 +6,7 @@
  */
 #import <TitaniumKit/TiModule.h>
 
-#ifdef USE_TI_CONTACTS
+#if defined(USE_TI_CONTACTS) && (!TARGET_OS_MACCATALYST || IS_SDK_IOS_14)
```

Comment:
I see that TARGET_OS_MACCATALYST is getting used. I guess it is not removed intentionally to save time. I think it is fine.

Reply:
Contacts API is available in Mac Catalyst 14+. So I intentionally guarded here like this.